pls_cli/please.py

```diff
@@ -131,6 +131,70 @@ def quotes(show: bool = True) -> None:
     )
 
 
+@app.command('language', rich_help_panel='Utils and Configs')
```

Comment:
Here I suggest to using enum, like this:
https://typer.tiangolo.com/tutorial/parameter-types/enum/